_data/sites.json

```
@@ -4916,6 +4916,16 @@
         ]
     },
 
+    {
+        "name": "Musescore",
+        "url": "https://musescore.com/user/settings/profile",
+        "difficulty": "easy",
+        "notes": "Click on 'Delete account' at the left of the page.",
+        "domains": [
+            "musescore.com",
+            "musescore.org"
+        ]
+    },
```

Now this is missing the newline afterward. Please add a line (with no trailing space) following this block.

```suggestion
    },

```

---

This might have been because the suggestion in the previous comment wasn't formatted correctly--apologies for that.